WultraMobileTokenSDK/Oidc/Utils/WMTOidcUtils.swift

```diff
@@ -0,0 +1,123 @@
+//
+// Copyright 2025 Wultra s.r.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions
+// and limitations under the License.
+//
+
+import Foundation
+
+/// Utility class for OIDC
+public class WMTOidcUtils {
+    
+    /// Creates PKCE codes, returning a Result wrapping the codes.
+    public static func createPKCE(dataLength: Int) throws -> WMTPKCECodes {
+        let minLength: Int = 32 // min is 32-octet sequence == Base64 43 URL safe characters
```

Comment:
the the min and max should be static let in the util class (to act as a compile constant). It's a recommended pattern rather than having it inside a function.